packages/formatter-summary/tests/tests.ts

```diff
@@ -13,25 +18,23 @@ import SummaryFormatter from '../src/formatter';
 import * as problems from './fixtures/list-of-problems';
 
 test.beforeEach((t) => {
-    sinon.spy(logging, 'log');
-
-    t.context.logger = logging;
+    t.context.loggingLogSpy = sinon.spy(logging, 'log');
 });
 
 test.afterEach.always((t) => {
-    t.context.logger.log.restore();
+    t.context.loggingLogSpy.restore();
 });
 
-test(`Summary formatter doesn't print anything if no values`, (t) => {
+test.serial(`Summary formatter doesn't print anything if no values`, (t) => {
```

Comment:
Was it a bug that this wasn't `serial` before?

Reply:
for some reason, it was working before without the serial, but now, with all the changes, it is failing without `.serial`

Reply:
Honestly it doesn't look like it should have worked reliably before. We may have been on the lucky side of a race condition (or other unexpected ava behavior that kept it from being a race condition). Makes sense to be `serial` now.

Reply:
Yes, because we are stubbing properties in an object share for both tests.